### PR TITLE
Fix navigation from dashboard to details page and details api

### PR DESCRIPTION
**Pull request by imazizbohra**

### CODE REVIEW

File: src/utils/api.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import axios from 'axios';
 
-const API_BASE_URL = 'https://api.example.com'; // Replace with your actual API base URL
+const API_BASE_URL = 'http://localhost:4200/api'; // Replace with your actual API base URL
```
Comment (sushilkundu143) [inline src/utils/api.js]: Can you move all the variable into .env @iamazizbohra

File: vite.config.js [hunk 2]
```diff
@@ -8,6 +8,6 @@ export default defineConfig({
     react(),
     tailwindcss()
   ],
-  base: '/webtrendmonitor_frontend/',
+  // base: '/webtrendmonitor_frontend/',
```
Comment (sushilkundu143) [inline vite.config.js]: uncomment this line. It will allow the deployment into github. @iamazizbohra

Comment (sushilkundu143) [reply vite.config.js]: Call server API's endpoints. Server APIs are deployed. @iamazizbohra

Comment (sushilkundu143) [reply vite.config.js]: https://website-hackathon-backend.vercel.app/ - server api base path @iamazizbohra

Comment (sushilkundu143) [reply vite.config.js]: https://website-hackathon-backend.vercel.app/api/publish-build?buildId=123456&sitemap=https://example.com/sitemap.xml

Try this get API for verify @iamazizbohra

Comment (imazizbohra) [reply vite.config.js]: ok

**Review by sushilkundu143 — APPROVED** [review]
Looks good to me.

**Pull request merged**